chat-commands.js

```diff
@@ -2921,13 +2921,17 @@ exports.commands = {
 		if (!user.hasConsoleAccess(connection)) {
 			return this.errorReply("/bash - Access denied.");
 		}
-
+		if (!target) {
+			return this.parse('/help bash');
+		}
 		connection.sendTo(room, "$ " + target);
 		let exec = require('child_process').exec;
 		exec(target, (error, stdout, stderr) => {
 			connection.sendTo(room, ("" + stdout + stderr));
 		});
 	},
+	bashhelp:["/bash [argument] - Carries out execution by running a child process",
+	 	  "Requires console access."],
```

Comment:
We have always stayed away from adding help entries for sysadmin-level commands. I don't see why we should depart from that practice now.